src/time/Storage.cpp

```diff
@@ -134,6 +134,15 @@ void Storage::trimAfter(double time)
   _bspline.reset();
 }
 
+void Storage::trimAfterIncluding(double time)
```

Comment:
Can you add a unit test for this function in `StorageTest.cpp`?

Reply:
Ah, I forgot to remove the function.

I realized at some point that `clear()` is all we need.